src/sentry/dynamic_sampling/models/transaction_adjustment_model.py

```diff
@@ -0,0 +1,190 @@
+from collections import namedtuple
+from copy import copy
+from typing import List, Mapping, MutableMapping, Tuple
+
+AdjustedSampleRate = namedtuple("AdjustedSampleRate", "explicit_rates, global_rate")
```

Comment:
I have a very similar idea, but I'm using a data class and some methods inside:
```
@dataclass
class AdjustedModel:
    """
    Model which can adjust sample rate per project inside ORG based on this new counter metric from Relay.
    """

    projects: List[Project]
    fidelity_rate: float

    @property
    def adjust_sample_rates(self):
        if len(self.projects) < 2:
            return self.projects
```

I think we can synchronize our codebases.

Reply:
here you can find https://github.com/getsentry/sentry/blob/c757357b016d7d32d1ea3c767a591ea3ed7838c7/src/sentry/dynamic_sampling/models/adjustment_models.py#L17-L34